content/en/Integrations/IntegrationBuilder/How to Guides/add-pentest-title-to-jira-issue-summary.md

```diff
@@ -0,0 +1,80 @@
+---
+title: "Add Pentest Title to Jira Issue Summary"
+linkTitle: "Add Pentest Title to Jira Issue Summary"
+weight: 70
+---
+
+{{% pageinfo %}}
+
+This guide is designed to help [**Integration Builder**](/integrations/integrationbuilder/) users customize the summary field of 
```

Comment:
I am reading the content and can't understand:
1. Why is the "Add Pentest Title to Jira Issue Summary" only showing the error handling branch and stopping the job? It looks more about handling a job error than adding a pentest title in general.

Reply:
Steps 13 and 14 show how to actually add the pentest title to the Jira issue summary.

Setting up the error handling __is__ most of the work in this guide since we have to fetch pentest data in order to add the pentest title to the issue summary.

As per this (https://github.com/cobalthq/cobalt-product-public-docs/pull/567#discussion_r1827317239) thread, we will move the error handling steps to a separate guide in the future.